### PR TITLE
ATS-985 - Upgrade LibreOffice to 7.2.5

### PR DESCRIPTION
**Pull request by evasques**

### CODE REVIEW

**Review by jottley — APPROVED** [review]

File: pom.xml [hunk 1]
```diff
@@ -369,7 +369,7 @@
                             <IMAGEMAGICK_EXE>/usr/bin/convert</IMAGEMAGICK_EXE>
                             <IMAGEMAGICK_DYN>/usr/lib64/ImageMagick-7.0.10/lib</IMAGEMAGICK_DYN>
                             <IMAGEMAGICK_ROOT>/usr/lib64/ImageMagick-7.0.10</IMAGEMAGICK_ROOT>
```
Comment (montgolfiere) [inline pom.xml:371]: Also this ?

**Comment by montgolfiere** [comment]
Thanks @evasques for taking this on.

It looks like endings have changed (in amongst other changes) ? 

See: https://github.com/Alfresco/alfresco-transform-core/pull/568
Workaround: https://github.com/Alfresco/alfresco-transform-core/pull/568?w=1

**Review by montgolfiere — CHANGES_REQUESTED** [review]
See two comments (re: line endings and some remaining references to 7.0.6).

File: pom.xml (hunk 1 above)
Comment (evasques) [reply pom.xml:371]: These are for imagemagik, not LO

**Review by stevenblair-al — APPROVED** [review]
great work and very glad you were able to do this so quickly Eva 🎉

File: pom.xml (hunk 1 above)
Comment (montgolfiere) [reply pom.xml:371]: Ah, sorry - my mistake (I just saw "7.0"). Busy day 🤦

**Review by montgolfiere — APPROVED** [review]
LGTM. Thanks @evasques 

Once released (eg. T-Core 2.6.0) we should also raise PRs to promote that version throughout other applicable projects (to run any additional e2e integrations tests, eg. such as Repo Renditions).

**Pull request merged**